### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by rlboyle** [community/community#129243] [Copilot Conversations]
### Select Topic Area

General

### Body

I sent in my application for the student developer pack about a month ago and haven't gotten anything back about it. I didn't even get a confirmation email saying I applied. Does anyone have any experience with this?

**Comment by hamburnyog** [comment]
Hi @rlboyle have you done all of this?

![image](https://github.com/user-attachments/assets/8f2ebd33-85be-4465-84b7-8b7d94edb9ae)
